auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java

```diff
@@ -224,9 +222,7 @@ public void finish(int resultCode, Intent resultIntent) {
         List<String> accounts = new ArrayList<>();
         for (AuthUI.IdpConfig idpConfig : getFlowParams().providerInfo) {
             @AuthUI.SupportedProvider String providerId = idpConfig.getProviderId();
-            if (providerId.equals(GoogleAuthProvider.PROVIDER_ID)
-                    || providerId.equals(FacebookAuthProvider.PROVIDER_ID)
-                    || providerId.equals(TwitterAuthProvider.PROVIDER_ID)) {
+            if (providerId.equals(GoogleAuthProvider.PROVIDER_ID)) {
```

Comment:
nit: can we change the name of this method to be "getCredentialAccountTypes" or something that implies it has to do with SmartLock?

Reply:
👍